format/spec.md

```diff
@@ -1239,6 +1247,9 @@ When reading an `unknown` column, any corresponding column must be ignored and r
 | **`struct`**       | `struct`            |                                                      |                                                                                         |
 | **`list`**         | `array`             |                                                      |                                                                                         |
 | **`map`**          | `map`               |                                                      |                                                                                         |
+| **`geometry`**     | `binary`            |                                                      | WKB format, see [Appendix G](#appendix-g-geospatial-notes).                                                      |
+| **`geography`**    | `binary`            |                                                      | WKB format, see [Appendix G](#appendix-g-geospatial-notes).                                                      |
```

Comment:
I think that ORC will require a type attribute like `uuid` and `fixed`. Looks like it should be `iceberg.binary-type` = `GEOMETRY` and `GEOGRAPHY`.